pkg/qemu/qemuconst/qemuconst.go

```diff
@@ -0,0 +1,6 @@
+package qemuconst
+
+const (
+	SlirpMACAddress  = "22:11:11:11:11:11"
```

Comment:
Maybe we should hash SlirpMACAddress too, but can be another PR.

Reply:
We could, but does it matter? Isn't it always on a network by itself?

Reply:
Some apps may want eth0 MAC to be unique.

Reply:
Ok, I'll add it to this PR.

Is the name `eth0` fine, or should it be something like `slirp` instead?

I think I prefer to always have an `eth0` interface though; it feels more traditional. 😄

Reply:
eth0 looks good to me

Reply:
And any suggestion about where to put `qemuconst.SlirpNICName`? It feels a little silly to have a separate package just for one constant.

Reply:
>  It feels a little silly to have a separate package just for one constant.

I think this is fine

Reply:
> Maybe we should hash SlirpMACAddress too

Implemented in 5f4d187.

I'm slightly worried that the SlirpMACAddress is calculated in 2 different places, and that if only one of them is updated later, the addresses would no longer match. Not sure how to avoid this though.